src/diffusers/models/embeddings.py

```diff
@@ -1080,6 +1132,39 @@ def forward(self, timestep, guidance, pooled_projection):
         return conditioning
 
 
+class CogView3CombinedTimestepConditionEmbeddings(nn.Module):
```

Comment:
Curious. 

How is this diffrent from the rest of conditioning embeddings that combine multiple conditions such as timestep, coordinates, etc.?